web3.js/src/loader.ts

```diff
@@ -160,7 +169,9 @@ export class Loader {
         {
           instruction: 0, // Load instruction
           offset,
-          bytes,
+          bytes: bytes as number[],
+          bytesLength: 0,
+          bytesLengthPadding: 0,
```

Comment:
Leaving these `undefined` essentially writes zero into the buffer so it's safe to explicitly supply a zero here. I verified this using `ts-node`, encoding once with undefined and once with zero. The same buffer was produced in both cases.